lib/web_ui/tool/unicode_sync_script.dart

```diff
@@ -50,28 +56,81 @@ class PropertyTuple {
   }
 }
 
-/// Usage (from the root of the project):
+final String codegenPath = path.join(
+  path.dirname(Platform.script.toFilePath()),
+  '../lib/src/engine/text',
+);
+final String wordBreakCodegen =
+    path.join(codegenPath, 'word_break_properties.dart');
+final String lineBreakCodegen =
+    path.join(codegenPath, 'line_break_properties.dart');
+
+/// Usage (from the root of the web_ui project).
+///
+/// To generate code for word break properties:
+/// ```
+/// dart tool/unicode_sync_script.dart -w <path/to/word/break/properties>
+/// ```
+///
+/// To generate code for line break properties:
+/// ```
+/// dart tool/unicode_sync_script.dart -l <path/to/line/break/properties>
+/// ```
+///
+/// To do a dry run, add the `-d` flag:
 ///
 /// ```
-/// dart tool/unicode_sync_script.dart <path/to/word/break/properties>
+/// dart tool/unicode_sync_script.dart -d ...
 /// ```
 ///
-/// This script parses the unicode word break properties(1) and generates Dart
+/// This script parses the unicode word/line break properties(1) and generates Dart
 /// code(2) that can perform lookups in the unicode ranges to find what property
 /// a letter has.
 ///
-/// (1) The properties file can be downloaded from:
-///     https://www.unicode.org/Public/11.0.0/ucd/auxiliary/WordBreakProperty.txt
+/// (1) The word break properties file can be downloaded from:
+///     https://www.unicode.org/Public/13.0.0/ucd/auxiliary/WordBreakProperty.txt
+///
+///     The line break properties file can be downloaded from:
+///     https://www.unicode.org/Public/13.0.0/ucd/LineBreak.txt
 ///
-/// (2) The codegen'd Dart file is located at:
-///     lib/src/text/word_break_properties.dart
+/// (2) The codegen'd Dart files is located at:
+///     lib/src/engine/text/word_break_properties.dart
+///     lib/src/engine/text/line_break_properties.dart
 void main(List<String> arguments) async {
-  final String propertiesFile = arguments[0];
```

Comment:
if (arguments.isEmpty) { print('Expecting properties file as argument'); exit...

Reply:
There is code below that checks and prints a message. I added to it the printing of auto-generated usage instructions from `argParser`.